.changeset/shiny-poems-occur.md

```diff
@@ -0,0 +1,3 @@
+---
+"@khanacademy/perseus": patch
```

Comment:
There's no changeset description. If you're making an empty changeset I think you need to delete this line; otherwise I think we need a description.

Reply:
Ah. I can manually delete this line here for sure. In the future, can I do `yarn changeset --empty` to create a true empty changeset from the start? Or do you do that another way?

Reply:
Yep.  `yarn changeset --empty` is how you do these empty change sets. :)